src/System.Private.CoreLib/shared/Interop/Windows/User32/Interop.LoadString.cs

```diff
@@ -3,14 +3,13 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Runtime.InteropServices;
-using System.Text;
 using Microsoft.Win32.SafeHandles;
 
 internal partial class Interop
 {
     internal partial class User32
     {
         [DllImport(Libraries.User32, SetLastError = true, EntryPoint = "LoadStringW", CharSet = CharSet.Unicode)]
-        internal static extern int LoadString(SafeLibraryHandle handle, int id, [Out] StringBuilder buffer, int bufferLength);
+        internal static unsafe extern int LoadString(SafeLibraryHandle hInstance, int uID, char* lpBuffer, int cchBufferMax);
```

Comment:
>uID [](start = 86, length = 3)

this looks weird to name it like that. in native side it is named uID because it is uint and here we define it as int.

Reply:
I'll change it to uint.  Our approach/style with P/Invokes in .NET Core is to keep everything as close to the native definition as possible.